spec/features/peep_metadata_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'sign_up_helper'
+
+# As a Maker
```

Comment:
I like how you've put the US's at the top, shows you're concise with covering all testing aspects